prvocisla.cpp

```diff
@@ -1,14 +1,14 @@
 #include <iostream>
-
 #include <string.h>
 
-
 using namespace std;
```

Comment:
toto je obecne spatny napad, co vim, ale pro tvoje potreby je to asi jedno: 
using namespace std;